builtInFunctions/saveUserName.go

```diff
@@ -78,7 +93,10 @@ func createCrossShardUserNameCall(
 ) (*vmcommon.VMOutput, error) {
 	vmOutput := &vmcommon.VMOutput{ReturnCode: vmcommon.Ok}
 	vmOutput.OutputAccounts = make(map[string]*vmcommon.OutputAccount)
-	setUserNameTxData := builtInFuncName + "@" + hex.EncodeToString(vmInput.Arguments[0])
+	setUserNameTxData := builtInFuncName
+	for _, arg := range vmInput.Arguments {
+		setUserNameTxData += "@" + hex.EncodeToString(arg)
+	}
```

Comment:
Is this backwards compatible for the old `saveUserName` builtInFunc?

Reply:
yes. it is.